messaging/Controllers/SteveController.cs

```diff
@@ -21,12 +21,12 @@ public class SteveController : BundlesController
 
         protected override IQueryable<OutgoingMessageItem> ExcludeRetrieved(IQueryable<OutgoingMessageItem> source)
         {
-            return source.Where(message => message.SteveRetrievedAt == null);
+            return source.Where(message => message.RetrievedAt == null);
         }
 
         protected override void MarkAsRetrieved(OutgoingMessageItem omi, DateTime retrieved)
```

Comment:
can be deleted because the override is now the same as the base class's implementation